s3tests_boto3/functional/test_s3.py

```diff
@@ -3229,73 +3229,53 @@ def _test_bucket_create_naming_good_long(length):
 @attr('fails_with_subdomain')
 @attr(resource='bucket')
 @attr(method='put')
-@attr(operation='create w/250 byte name')
+@attr(operation='create w/60 byte name')
 @attr(assertion='fails with subdomain')
-@attr('fails_on_aws') # <Error><Code>InvalidBucketName</Code><Message>The specified bucket is not valid.</Message>...</Error>
-def test_bucket_create_naming_good_long_250():
-    _test_bucket_create_naming_good_long(250)
+# @attr('fails_on_aws') # <Error><Code>InvalidBucketName</Code><Message>The specified bucket is not valid.</Message>...</Error>
```

Comment:
I am not sure whats the status of these testcases on aws (Ideally they should pass). Hence have commented them out for now. Please suggest.